train.py

```diff
@@ -85,7 +91,7 @@ def total_variation_regularization(x, beta=1):
 feature_s = vgg(Variable(style_b, volatile=True))
 gram_s = [gram_matrix(y) for y in feature_s]
 
-for epoch in range(n_epoch):
+for epoch in range(m_epoch, n_epoch):
```

Comment:
Yeah I think that would make sense.

I also noticed that. A good approach is to write out checkpoints every few thousand images.